### ISSUE TITLE
NTR: Folate absorption impaired (Triggers or responses: drugs or treatments)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9491]
New term request Folate absorption impaired (segal_110901192547): Tested with oral load of 5-formyltetrahydrofolate (5-formylTHF; Leucovorin®, folinic acid) or 5-methyltetrahydrofolate (5-methylTHF; 5-MTHF; Metafolin®)

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000258
- label: Impaired folate absorption
- parents: [Abnormality of folate metabolism (HP:0012335)](https://hpo.jax.org/app/browse/term/HP:0012335)
- pmid: PMID:11804211
- def: Reduced ability to absorb folate in the diet. This finding can be measured by a folic acid loading test in which blood concentration of folate is measured after oral administration of folic acid.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9491
- synonyms:
